runtime/battery-station/src/lib.rs

```diff
@@ -118,6 +122,7 @@ impl Contains<Call> for IsCallable {
                         dispute_mechanism: Court | SimpleDisputes,
                         ..
                     } => false,
+                    start_global_dispute { market_id: _ } => false,
```

Comment:
Mhhh, good questions. Currently we allow only one dispute on the main net and [six disputes on battery station](https://github.com/zeitgeistpm/zeitgeist/blob/294c66c4d070171fbe4389fbb63b1608e7df3ba8/runtime/battery-station/src/parameters.rs#L153), although I thought it wouldn't make much sense to allow multiple disputes while there is an authority who makes the final shots. 

However, my current implementation of `start_global_dispute` for `global-disputes` needs `MaxDisputes` to allow starting a global dispute. @maltekliemann , would it make sense to start global disputes with an authorized market dispute mechanism to start testing it? When should the global dispute start in that situation? 

`global-disputes` is currently no dispute mechanism anymore, because it should be the last instance, when a market dispute mechanism fails. I could feel comfortable with starting a global dispute, when 2 disputes are present to even override the authority with the result of the global dispute, but should be really meant to be for a test net, not for a main net first (as you proposed). I could live with that, but it really means, that the authority looses control once a global dispute started.

Reply:
Can we just simply attach it to current simple disputes? The idea being that people submit their disputes with an increasing bond as it is currently in simple disputes. In the end, if the highest bond was submitted to an outcome, we use the api provided by global disputes to pass the outcome owners and their dispute bonds to the global disputes crate. Then global disputes takes over and finally punished all wrong outcome providers and rewards correct outcome providers. I think having simple disputes coupled with global disputes makes the disputers much more careful with providing the dispute outcomes on simple disputes, because they know eventually all token holders will decide (and they risk losing it all). From my perspective, simple disputes becomes meaningful again with global disputes - global disputes will probably more rarely be triggered because users who dispute outcomes in simple disputes have the fear of losing it all through the uncertainty of the global dispute.

Reply:
That sounds like a plan. @maltekliemann Do you agree? Sure, we have the negative effect, that "poor Paul" can't effort to dispute, because the bond could be too high. But the "rich Richard" is incentivised to get the disputor bonds, if the outcome seems wrong.

Reply:
923c2812b6099b00fb10222365551bb8a60896bb